web/ts/watch.ts

```diff
@@ -30,14 +58,25 @@ export function initChatScrollListener() {
     });
 }
 
-export function scrollChatIfNeeded() {
+export function shouldScroll(): boolean {
+    if (orderByLikes) {
+        return false; // only scroll if sorting by time
+    }
     const c = document.getElementById("chatBox");
-    // 150px grace offset to avoid showing message when close to bottom
-    if (c.scrollHeight - c.scrollTop <= c.offsetHeight + 150) {
-        c.scrollTop = c.scrollHeight;
-    } else {
-        window.dispatchEvent(new CustomEvent("messageindicator", { detail: { show: true } }));
+    return c.scrollHeight - c.scrollTop <= c.offsetHeight;
+}
+
+function showNewMessageIndicator() {
+    window.dispatchEvent(new CustomEvent("messageindicator", { detail: { show: true } }));
+}
+
+export function scrollChat() {
+    console.log("scrollChatIfNeeded");
```

Comment:
Just marking this in case it was forgotten to remove it. Otherwise it's fine.

Reply:
Yup, thanks :)